simulation_ws/src/test_nodes/nodes/docking_test.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+
+import rospy
+import rostest
+import time
+import os
+import unittest
+import tf
+
+from math import hypot
+from std_msgs.msg import Bool
+from geometry_msgs.msg import PoseStamped
+from rosgraph_msgs.msg import Clock
+from nav_msgs.msg import Odometry
+from std_srvs.srv import Trigger
+
+from robomaker_simulation_msgs.msg import Tag
+from robomaker_simulation_msgs.srv import Cancel, AddTags
+
+class DockingTest(unittest.TestCase):
+
+    def runTest(self):
+        # Start the docking test
+        self.test_docking()
+
+    def setUp(self):
+        self.test_name = "Robot_Monitoring_Tests_" + \
+            str(time.time()).split(".", 1)[0]
```

Comment:
Write a function and call it from here:
```
def get_time(self):
    return rospy.Time.now().secs
    
self.test_name = "Docking_Test_" + str(self.get_time())
```
It'll return the same, but using sim time